### ISSUE TITLE
Genie is wrongly configuring WSLInterop binfmt_misc

### ISSUE THREAD

**Issue by NyaMisty** [arkane-systems/genie#267]
**Windows version (build number):**
10.0.22598.200

**Linux distribution:**
Ubuntu jammy

**Kernel version:**
5.10.102.1-microsoft-standard-WSL2

**Genie version:**
genie 2.3

**Describe the bug**
Running `genie -i` would cause Windows exe failed to find first parameter.

Running `git.exe`: empty output & exitcode 1
Running `git.exe status`: prints git help
Running `git.exe status status`: prints same with original `git.exe status`

**To Reproduce**
1. genie -i
2. git.exe status

**Expected behavior**
Apprarently

**Additional context**
This problem is actually caused by wrongly configured `/usr/lib/binfmt.d/WSLInterop.conf`:
```
:WSLInterop:M::MZ::/init:F
```
But the original `cat /proc/sys/fs/binfmt_misc/WSLInterop` is:
```
enabled
interpreter /tools/init
flags: PF
offset 0
magic 4d5a
```

The missing P is the problem. Changing F to PF in `WSLInterop.conf` would solve this issue, but I can't confirm whether other Windows version have the same situation.

**Comment by cerebrate** [comment]
Fix added to 2.4 release. Thanks for catching this!

**Comment by gnw3** [comment]
My WSL Debian did not have a `/usr/lib/binfmt.d/WSLInterop.conf`,  I created on and tried it with and without the "P" flag. Debian required th e`P`.  WSLg works with:
```
 % cat /usr/lib/binfmt.d/WSLInterop.conf
:WSLInterop:M::MZ::/init:PF
```
